config.yaml

```diff
@@ -45,8 +45,10 @@ params:
             link: https://headlamp.dev/docs/latest/installation/desktop/
           - title: Plugin development
             link: https://headlamp.dev/docs/latest/development/plugins/
+          - title: Frequently Asked Questions
```

Comment:
I'd prefer `F.A.Q.`. I think it more quickly catches the eye.
Up to you @blixtra

Reply:
done!